docs/source/contributing/developing.adoc

```diff
@@ -15,7 +15,7 @@ The following sections describe how to build and test Minishift.
 == Prerequisites
 
 - Git
-- A recent Go distribution (>1.7)
+- A recent Go distribution (>1.8)
```

Comment:
@praveenkumar @hferentschik I think we should keep it (>1.7) as it still works with 1.7.

Reply:
I think we should raise bar here as well. I don't want to have to chase issues due to someone using an older Go version. Especially if we make some Make changes uses some 1.8 features.

Reply:
I do agree, let's not use an older version on the doc side.

Reply:
Ok, then we should mark the version specifically 1.8 because with current change means it will work with 1.9 and future version i.e. Go distribution version 1.8` or at least `Go distribution (>= 1.8)`.

Reply:
+1 for >= 1.8